### ISSUE TITLE
Cannot read property 'charCodeAt' of undefined, atob is not defined

### ISSUE THREAD

**Issue by LittleWhiteYA** [codemanki/cloudscraper#152]
```
Unhandled rejection ParserError: 
### Cloudflare may have changed their technique, or there may be a bug.
### Bug Reports: https://github.com/codemanki/cloudscraper/issues
### Check the detailed exception message that follows for the cause.

TypeError: Challenge evaluation failed: Cannot read property 'charCodeAt' of undefined
    at solveChallenge (xxx/node_modules/cloudscraper/index.js:248:21)
    at Timeout._onTimeout (xxx/node_modules/cloudscraper/index.js:179:7)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)

Unhandled rejection ParserError: 
### Cloudflare may have changed their technique, or there may be a bug.
### Bug Reports: https://github.com/codemanki/cloudscraper/issues
### Check the detailed exception message that follows for the cause.

ReferenceError: Challenge evaluation failed: atob is not defined
    at solveChallenge (xxx/node_modules/cloudscraper/index.js:248:21)
    at Timeout._onTimeout (xxx/node_modules/cloudscraper/index.js:179:7)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
```

version:
- 3.2.0 also happens.

Hi, I got these errors for few days.
Not sure how to solve it ...
has the challenge been updated?

**Comment by pro-src** [comment]
@LittleWhiteYA The error in question does seem to indicate that your facing a unique challenge. Unique in that I can't reproduce it with sites that have IUAM enabled. I can't determine the source of the problem unless you provide the source of the page that contains the challenge. Please copy and paste the challenge page's HTML into a pastebin and share the link.

See how to view the page's source: https://www.wikihow.com/View-Source-Code
See how to create a pastebin: https://openedx.atlassian.net/wiki/spaces/OXA/pages/158076804/How+to+use+a+pastebin

You might also try clearing your browser's history to get the challenge page.

**Comment by pro-src** [comment]
@LittleWhiteYA Nvm, I overlooked it. The challenge definitely has been updated.

**Comment by pro-src** [comment]
@LittleWhiteYA Thanks for reporting this! The fix in #153 is pending review and a new version will be released soon.

**Comment by Be4t5** [comment]
> @LittleWhiteYA Thanks for reporting this! The fix in #153 is pending review and a new version will be released soon.

Thanks for the fix!

**Comment by codemanki** [comment]
This issue has been fixed ( thank you @pro-src for saving the day. again :) ). `3.3.0` is now published
